doc/WebSite/Dapper-Usage.html

```diff
@@ -0,0 +1,187 @@
+        Map(x => x.CreatorUser).Ignore();
+        Map(x => x.LastModifierUser).Ignore();
+        Map(x => x.FullName).Ignore();
+        Map(x => x.Tokens).Ignore();
+
+        AutoMap();
+    }
+}</pre>
+			
+			<p>After define mappings, you need to register them in module initialize method.</p>
+						<pre lang="cs">[DependsOn(
+     typeof(AbpEntityFrameworkModule),
+     typeof(AbpDapperModule)
+)]
+public class SampleApplicationModule : AbpModule
+{
+    public override void Initialize()
+    {
+        IocManager.RegisterAssemblyByConvention(typeof(SampleApplicationModule).GetAssembly());
+        <strong>DapperExtensions.DapperExtensions.SetMappingAssemblies(new List<Assembly> { typeof(SampleApplicationModule).GetAssembly() });</strong>
+    }
+}
+			</pre>
+			
+			<h3 id="DocUsage">Usage</h3>
+
+			
+			<p>After registering <strong>AbpDapperModule</strong>, you can use it like traditinal ABP repositories like following.</p>
+			<pre lang="cs">public class SomeDomainService : ITransientDependency
+{
+    private readonly IDapperRepository&lt;Animal&gt; _animalDapperRepository;
+    private readonly IRepository&lt;Animal&gt; _animalRepository;
+    private readonly IDapperRepository&lt;Person&gt; _personDapperRepository;
+    private readonly IRepository&lt;Person&gt; _personRepository;
+    private readonly IUnitOfWorkManager _unitOfWorkManager;
+
+    public SomeDomainService(
+        IUnitOfWorkManager unitOfWorkManager,
+        IRepository&lt;Person&gt; personRepository,
+        IRepository&lt;Animal&gt; animalRepository,
+        IDapperRepository&lt;Person&gt; personDapperRepository,
+        IDapperRepository&lt;Animal&gt; animalDapperRepository)
+    {
+        _unitOfWorkManager = unitOfWorkManager;
+        _personRepository = personRepository;
+        _animalRepository = animalRepository;
+        _personDapperRepository = personDapperRepository;
+        _animalDapperRepository = animalDapperRepository;
+    }
+
+    public void DoSomeStuff()
+    {
+        using (IUnitOfWorkCompleteHandle uow = _unitOfWorkManager.Begin())
+        {
+            _personRepository.Insert(new Person("John"));
+            _personRepository.Insert(new Person("Bread"));
+
+            _animalRepository.Insert(new Animal("Bird"));
+            _animalRepository.Insert(new Animal("Cat"));
+
+            _unitOfWorkManager.Current.SaveChanges();
+
+            Animal animal = _animalRepository.FirstOrDefault(x => x.Name == "Bird");
+
+            Person person = _personDapperRepository.Get(1);
+            int personCount = _personDapperRepository.Count(x => x.Name == "John");
+            List&lt;Animal&gt; persons = _animalDapperRepository.GetList(x => x.Name.StartsWith("O")).ToList();
+
+            uow.Complete();
+        }
+    }
+}
+</pre>
+			<p>You can use both EF repositories and Dapper repositories at the same time.</p>
+			
+			<h3 id="DocUseSql">SQL Statements</h3>
+			<p>You can write pure SQL statements, if you want.</p>
```

Comment:
Move this samples to above so no need to this section. Also, each letter of each word of titles should be UPPER case.